src/ViewRendererBundle/Command/CompileLessCommand.php

```diff
@@ -64,7 +70,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $portalList = TdbCmsPortalList::GetList();
         $minifyCss = true === $input->getOption('minify-css');
         while ($portal = $portalList->Next()) {
+            $this->themeService->setOverrideTheme($portal->GetFieldPkgCmsTheme());
             $css = $this->lessCompiler->getGeneratedCssForPortal($portal, $minifyCss);
+            $this->themeService->setOverrideTheme(null);
+
```

Comment:
The command must also compile less for the backend - as an additional step

Reply:
There is no less in the backend.

(no loading, no location)

Reply:
A theme supports less, does it not? So we should compile it here (if the theme has less). Even if it is not currently loaded. In fact, we should probably load it too (if that is easily done)

Reply:
At the moment a portal supports less.

Everything from TPkgViewRendererLessCompiler::getGeneratedCssForPortal() onwards needs a portal.

This would be in any case a new feature / issue.